Classes/RenderingDispatcher.php

```diff
@@ -52,13 +52,14 @@ class RenderingDispatcher
     protected $requestBuilder;
 
     /**
-     * @var array
+     * @var string[]
      */
     protected $renderer = array();
 
     /**
      * @param RequestBuilder $requestBuilder
-     * @param array $renderer
+     *
+     * @param string[] $renderer
```

Comment:
that is fine btw. It give additional information of what the types are in the array